docs/usage/control-plane-trusted-certificates.md

```diff
@@ -0,0 +1,48 @@
+# Control Plane Trusted Certificates
+Shoot clusters are composed of several control plane components deployed by the Gardener and corresponding extensions.
+
+Some components are exposed via `ingress` resources which make them addressable under the HTTPS protocol.
+
+Examples:
+- Alertmanager
+- Grafana operator & user
+- Prometheus
+- Kibana
+
+Gardener generates the backing TLS certificates which are signed by the `Kubernets` cluster CA (shoot's cluster CA) by default.
+
+Unlike with a self-contained Kubeconfig file, common internet browsers or operating systems don't trust a shoot's cluster CA and adding it as a trusted root is often undesired in enterprise environments.
+
+Therefore, Gardener operators can predefine trusted certificates under which the mentioned endpoints will be served instead.
+
+## Register a trusted wildcard certificate
+Since control plane components are published under the ingress domain (`seed.spec.dns.ingressDomain`) a wildcard certificate is required.
```

Comment:
```suggestion
Since control plane components are published under the ingress domain (`core.gardener.cloud/v1beta1.Seed.spec.dns.ingressDomain`) a wildcard certificate is required.
```